src/current/v24.1/molt-fetch.md

```diff
@@ -91,9 +89,34 @@ Complete the following items before using MOLT Fetch:
 			~~~
 
 		- Ensure the Google Cloud Storage bucket is created and accessible to CockroachDB.
-	
+
+## Best practices
+
+- To prevent connections from prematurely terminating during data export, set the following to high values on the source database:
+
+	- **Maximum allowed number of connections.** MOLT Fetch can export data across multiple connections. The number of connections it will create is the number of shards ([`--export-concurrency`](#global-flags)) multiplied by the number of tables ([`--table-concurrency`](#global-flags)) being exported concurrently.
```

Comment:
It looks weird to me that these have periods. I think maybe they should have colons after the bold instead.